nilearn/tests/test_niimg_conversions.py

```diff
@@ -341,6 +343,23 @@ def test_iter_check_niimgs():
                        _utils.check_niimg(img_2_4d).get_data())
 
 
+def _check_memory(list_img_3d):
+    # We intentionally add an offset of memory usage to avoid non trustable
+    # measures with memory_profiler.
+    mem_offset = b'a' * 100 * 1024 ** 2
+    list(_iter_check_niimg(list_img_3d))
+    del mem_offset
```

Comment:
Codacy disliked the previous code, if I remove the `del` line there's a flake8 issue on travis.